bootstrapper/internal/kubernetes/kubernetes.go

```diff
@@ -301,6 +331,15 @@ func (k *KubeWrapper) StartKubelet() error {
 		return fmt.Errorf("starting kubelet: %w", err)
 	}
 
+	k.log.Info("Prioritizing etcd I/O")
+	err := k.etcdIOPrioritizer.PrioritizeIO()
+	if errors.Is(err, etcdio.ErrNoEtcdProcess) {
+		k.log.Warn("Skipping etcd I/O prioritization as etcd process is not running. " +
+			"This is expected if this node is a non-control-plane node.")
+	} else if err != nil {
+		return fmt.Errorf("prioritizing etcd I/O: %w", err)
+	}
```

Comment:
```suggestion
	if peerRole == role.ControlPlane {
		k.log.Info("Prioritizing etcd I/O")
		if err := k.etcdIOPrioritizer.PrioritizeIO(); err != nil {
			return fmt.Errorf("prioritizing etcd I/O: %w", err)
		}
	}
```
Same thing here

Reply:
We don't know about our role here. I think it's easier to check for the error in this case